src/globals/Calc.ts

```diff
@@ -179,7 +205,21 @@ interface CalcPrivate {
         killWorker: () => void;
       };
     };
-    listModel: unknown;
+    listModel: {
+      selectedItemMap: Record<number, boolean | undefined>;
+
+      // add properties as needed
+      __itemModelArray: {
+        id: string;
+        controller: Calc["controller"];
```

Comment:
I'm guessing `controller` here isn't used since you can just do `Calc.controller`.

Reply:
Yeah, it's not being used. I just removed it and got no errors.